templates/registration/login.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% block content %}
+  <h3>Login</h3>
+  <form action="{% url "taxi:login" %}" method="post">
```

Comment:
Inconsistent use of quotation marks in the template tag. Consider using single quotes consistently for URL template tags, like `{% url 'taxi:login' %}`.